app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -6,8 +6,8 @@ import { Container } from "neetoui/layouts";
 import { NOTES_DATA } from "./constants";
 import DeleteAlert from "./DeleteAlert";
 import Note from "./Note";
-import NotesHeader from "./NotesHeader";
 import NewNotePane from "./Pane/Create";
+import Header from "./TitleBar";
```

Comment:
I don't understand the logic behind this naming. How do you associate TitleBar to Header? And most important question here is why not name the component and file as `Header` itself? Why name it `TitleBar` and then import it under an alias?

Reply:
Since the Header component already had a Header component from neetoui/layouts naming the main component also as Header was giving an error.

Reply:
@chiragbagde Read: https://academy.bigbinary.com/learn-react/pages/miscellaneous-tips#:~:text=Importing%20under%20an%20alias